spikecomparison/multisortingcomparison.py

```diff
@@ -198,44 +227,62 @@ def _do_agreement_matrix(self, minimum_matching=0):
                     agreement_matrix[u_i, sort_name] = sorting_agr.get_unit_property(unit, 'avg_agreement')
         return agreement_matrix
 
-    def plot_agreement(self, minimum_matching=0):
-        import matplotlib.pylab as plt
-        sorted_name_list = sorted(self.name_list)
-        sorting_agr = AgreementSortingExtractor(self, minimum_matching)
-        unit_ids = sorting_agr.get_unit_ids()
-        agreement_matrix = self._do_agreement_matrix(minimum_matching)
-
-        fig, ax = plt.subplots()
-        # Using matshow here just because it sets the ticks up nicely. imshow is faster.
-        ax.matshow(agreement_matrix, cmap='Greens')
-
-        # Major ticks
-        ax.set_xticks(np.arange(0, len(sorted_name_list)))
-        ax.set_yticks(np.arange(0, len(unit_ids)))
-        ax.xaxis.tick_bottom()
-        # Labels for major ticks
-        ax.set_xticklabels(sorted_name_list, fontsize=12)
-        ax.set_yticklabels(unit_ids, fontsize=12)
-
-        ax.set_xlabel('Sorters', fontsize=15)
-        ax.set_ylabel('Units', fontsize=20)
-
-        return ax
+    def _remove_duplicate_edges(self):
```

Comment:
I would add a comments to explain why we have this function.
like one or two sentence.